emojivoto-web-app/js/setup_dev_env.sh

```diff
@@ -198,10 +198,11 @@ connect_local_dev_env_to_remote() {
     telepresence quit
     telepresence helm upgrade --team-mode
     telepresence login --apikey=${AMBASSADOR_API_KEY}
-    telepresence connect
+    telepresence quit -s
```

Comment:
In line `198` we already have the `telepresence quit` it may make sense to just include the flag `-s` to that command. Or do we need to do quit twice?

Reply:
I thing I need to call it again because telepresence helm upgrade start the user daemon, so if you don't do it, the cli will complain that a local daemon is running so it can't stant the docker one.